runtime/Runtest/intrinsics.c

```diff
@@ -101,7 +101,12 @@ void klee_make_symbolic(void *array, size_t nbytes, const char *name) {
       exit(1);
     }
   }
-
+  if (testPosition == 0) {
+    char *testPositionFromEnv = getenv("TEST_POSITION");
```

Comment:
The environment variable should probably get a less generic name (e.g.m `KLEE_TEST_POSITION`).